packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1370,6 +1378,21 @@ test('transfer vault', async t => {
     'transfer closed old notifier',
   );
 
+  t.deepEqual(
```

Comment:
I think the contract should publish a validation function that clients can use to check a purported transfer invitation. The contract would know that the recipient should check the instance, possibly governance, provide a minimum for locked collateral and a maximum debt, and know the allowed states. 
I don't think ad hoc checking of some aspects of the state is enough to qualify as legibility.

Reply:
Can you point me at an example of what you are thinking? It seems like the real thing should include patterns (e.g., Fred wants a vault worth at least X), which shouldn't be in this PR.

Reply:
Chris and I discussed. This should turn into support for creating a relevant `want` pattern once that's available.